### PR TITLE
feat(list-view): List View item hover state changes

### PR DESCRIPTION
**Pull request by joshmarnold**
Adding Overflow16 Icon.

### CODE REVIEW

**Comment by QingyuChai** [comment]
> Rename icon should be this asset: https://opensource.box.com/box-ui-elements/storybook/?path=/story/icon-fill-pencil16--pencil-16 Ellipsis icon should be replaced with this asset from design-assets: https://git.dev.box.net/Box/design-assets/pull/163)

Hi, I think it's preferred not to link internal git repos!

File: src/icons/iconTypes.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ export interface Icon extends TwoTonedIcon {
     /** A string describing the fill color for the icon */
     fillColor?: string;
     /** A number specifying the opacity of the icon */
+    isEnhancement?: boolean;
+    /** A number specifying the opacity of the icon */
```
Comment (QingyuChai) [inline src/icons/iconTypes.ts:21]: Could you update the prop description

**Comment by joshmarnold** [comment]
Closing PR for now to add new assets via internal tool.

**Review by tjuanitas — APPROVED** [review]
could we rename the PR title before this is merged. thanks!

**Pull request merged**